recipes/genera/meta.yaml

```diff
@@ -0,0 +1,42 @@
+package:
+  name: genera
+  version: "1.4.2"
+
+source:
+  - url: https://github.com/josuebarrera/GenEra/archive/refs/tags/v1.4.2.tar.gz
+    sha256: acc49698b68151550b825c73f1f03dd268da2c73ecc7ac4c55eccb8bec5b1a54
+  - url: https://github.com/caraweisman/abSENSE/archive/refs/tags/v1.0.0.tar.gz
+    sha256: a99090c10a66afae1fbad38d79d228199e9fbdae5173e71ad5d84b4c958dda54
+    folder: abSENSE
+
+build:
+  number: 0
+  run_exports:
+    - {{ pin_subpackage('genera', max_pin="x.x") }}
+
+requirements:
+  host:
+    - python =3.8.0
+  run:
+    - python =3.8.0
```

Comment:
please do not spin that strict, this is not how conda works. Strict pins can be used in environments.yml files, but packages should specify pins when they are really incompatible.